listenbrainz/model/playlist.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+from datetime import datetime
+
+from sqlalchemy.dialects.postgresql import UUID, JSONB
+
+from listenbrainz.model import db
+from listenbrainz.webserver.admin import AdminModelView
+
+
+class Playlist(db.Model):
+    __bind_key__ = 'timescale'
+    __tablename__ = "playlist"
+    __table_args__ = {"schema": "playlist"}
+
+    id = db.Column(db.Integer, autoincrement=True, primary_key=True)
+    mbid = db.Column(UUID(as_uuid=True), nullable=False, default=uuid.uuid4)
```

Comment:
similar to the comment above - in sqlalchemy does this `default` happen in python or in postgres? Given that we have a database-level default value, I'm not sure if we should _also_ have a python-level default here, which could cause confusion. What do you think?

Reply:
Right, this is a python default. This will only be useful if we want to add a playlist using admin UI. Should I remove it or make it a server side default?

Reply:
my intuition would be to disable adding in the admin interface, and remove the python default values.

Reply:
Done.